src/OSPSuite.UI/Views/Charts/ChartDisplayView.Designer.cs

```diff
@@ -1,257 +1,65 @@
-         this._StatusBar = new DevExpress.XtraBars.Bar();
-         this.barDockControlTop = new DevExpress.XtraBars.BarDockControl();
-         this.barDockControlBottom = new DevExpress.XtraBars.BarDockControl();
-         this.barDockControlLeft = new DevExpress.XtraBars.BarDockControl();
-         this.barDockControlRight = new DevExpress.XtraBars.BarDockControl();
-         this._chartControl = new UxChartControl(useDefaultPopupMechanism: false);
+         this._chartControl = new OSPSuite.UI.Controls.UxChartControl(useDefaultPopupMechanism:false);
          ((System.ComponentModel.ISupportInitialize)(this.errorProvider)).BeginInit();
-         ((System.ComponentModel.ISupportInitialize)(this._barManager)).BeginInit();
          ((System.ComponentModel.ISupportInitialize)(this._chartControl)).BeginInit();
-         this._chartControl.SuspendLayout();
          this.SuspendLayout();
          // 
-         // _barManager
-         // 
-         this._barManager.Bars.AddRange(new DevExpress.XtraBars.Bar[] {
-            this._toolBar,
-            this._menuBar,
-            this._StatusBar});
-         this._barManager.DockControls.Add(this.barDockControlTop);
-         this._barManager.DockControls.Add(this.barDockControlBottom);
-         this._barManager.DockControls.Add(this.barDockControlLeft);
-         this._barManager.DockControls.Add(this.barDockControlRight);
-         this._barManager.Form = this;
-         this._barManager.MaxItemId = 9;
-         // 
-         // _toolBar
-         // 
-         this._toolBar.BarName = "Tools";
-         this._toolBar.DockCol = 0;
-         this._toolBar.DockRow = 1;
-         this._toolBar.DockStyle = DevExpress.XtraBars.BarDockStyle.Top;
-         this._toolBar.Text = "Tools";
-         this._toolBar.Visible = false;
-         // 
-         // _menuBar
-         // 
-         this._menuBar.BarName = "Main menu";
-         this._menuBar.DockCol = 0;
-         this._menuBar.DockRow = 0;
-         this._menuBar.DockStyle = DevExpress.XtraBars.BarDockStyle.Top;
-         this._menuBar.FloatLocation = new System.Drawing.Point(132, 277);
-         this._menuBar.OptionsBar.MultiLine = true;
-         this._menuBar.OptionsBar.UseWholeRow = true;
-         this._menuBar.Text = "Main menu";
-         this._menuBar.Visible = false;
-         // 
-         // _StatusBar
-         // 
-         this._StatusBar.BarName = "Status bar";
-         this._StatusBar.CanDockStyle = DevExpress.XtraBars.BarCanDockStyle.Bottom;
-         this._StatusBar.DockCol = 0;
-         this._StatusBar.DockRow = 0;
-         this._StatusBar.DockStyle = DevExpress.XtraBars.BarDockStyle.Bottom;
-         this._StatusBar.OptionsBar.AllowQuickCustomization = false;
-         this._StatusBar.OptionsBar.DrawDragBorder = false;
-         this._StatusBar.OptionsBar.MultiLine = true;
-         this._StatusBar.OptionsBar.UseWholeRow = true;
-         this._StatusBar.Text = "Status bar";
```

Comment:
There was so much crap in the designer. Menu, bars, 10 titles!!! WTF? I just redraw the whole control